src/services/with-current-project.js

```diff
@@ -20,12 +24,15 @@ module.exports = async function withCurrentProject(config) {
       return { ...config, projectId: projects[0].id };
     }
 
+    const existingSpinner = spinner.isRunning();
+    if (existingSpinner) spinner.pause();
```

Comment:
[MISC] You could add a comment here to specify that this is intended for usage outside of a spinner context